author	Waldemar Brodkorb <wbx@openadk.org>	2024-05-08 10:08:08 +0200

futimesat: add missing header

diff --git a/libc/sysdeps/linux/common/futimesat.c b/libc/sysdeps/linux/common/futimesat.c
index fd19fea7c..bf36550dd 100644
--- a/libc/sysdeps/linux/common/futimesat.c
+++ b/libc/sysdeps/linux/common/futimesat.c
@@ -5,10 +5,11 @@
  *
  * Licensed under the LGPL v2.1, see the file COPYING.LIB in this tarball.
  */

 #include <sys/syscall.h>
+#include <sys/stat.h>
 #include <sys/time.h>

 #ifdef __NR_futimesat
 _syscall3(int, futimesat, int, fd, const char *, file, const struct timeval *, tvp)
 #elif defined __NR_utimensat
